src/components/autocomplete/Autocomplete.vue

```diff
@@ -27,6 +27,10 @@
                 v-show="isActive && (data.length > 0 || hasEmptySlot)"
                 ref="dropdown">
                 <div class="dropdown-content">
+                    <div
+                        class="dropdown-item">
```

Comment:
When there is no header slot you shouldn't show this element. Please add `v-if` using a computed prop as `hasHeaderSlot`.